README.md

```diff
@@ -21,10 +21,55 @@ other components of OpenShift via support for a baremetal platform type.
 
 # Instructions
 
-## Configuration
+## Preparation
+
+Considering that this is a new install on a clean OS, the next tasks should be performed prior the installation:
+
+1. Enable passwordless sudo for the current user
+
+    `echo "$USER  ALL=(ALL) NOPASSWD: ALL" >/etc/sudoers.d/${USER}`
+
+2. In case of RHEL, invoke `subscription-manager` in order to `register` and `attach` the subscription
+
+3. Install new packages
+
+    `sudo dnf update`
+    `sudo dnf install git make wget tmux jq`
+
+4. Clone the dev-scripts repository
+
+    `git clone https://github.com/openshift-metal3/dev-scripts`
+
+5. Create a config file
 
-Make a copy of the `config_example.sh` to `config_$USER.sh`, and set the
-`PULL_SECRET` variable to the secret obtained from cloud.openshift.com.
+    `cp config_example.sh config_$USER.sh`
+
+6. Configure secrets:
+   1. Download personal pull-secret file from https://cloud.redhat.com/openshift/install/pull-secret
+   2. Get the shared secret from a coworker
+   3. Combine the two files and add them to the user config
+
+    ```bash
+    # Form a JSON entry and add the shared secret to it
+    shared_secret='{"registry.svc.ci.openshift.org": {
+       "auth": "PLACE_SECRET_HERE" 
+    }}'
+    # Add the entry into the pull-secret file (assuming it's in the directory)
+    tmp_secret=$(cat pull-secret);echo $tmp_secret | jq --argjson secret $shared_secret '.["auths"] + $secret' >pull-secret
+    # Add the combined secret into the user config file
+    sed -i -e "/PULL_SECRET=''/ {" -e "s/''/'/" -e "r pull-secret" -e "a \\'" -e "}" config_$USER.sh
+    ```
+
+7. Configure dev-scripts working directory
+
+    By default, dev-scripts' working directory is set to `/opt/dev-scripts`.
+    Make sure that the filesystem has at least 80GB of free space: `df -h /`.
+    
+    Alternatively you may have a large `/home` filesystem.
+    The path can be overwritten with the `WORKING_DIR` environment variable in the user config file.
+    In case you change the directory, have it world-readable (`chmod 755`) and `chown`ed by `$USER`.
```

Comment:
It's not necessary to pre-create the directory, so you can e.g do `export WORKING_DIR=/home/dev-scripts` in the case where you have a large /home then it's created here:

https://github.com/openshift-metal3/dev-scripts/blob/master/common.sh#L279

So this could be reworded e.g "Alternatively you may have a large `/home` filesystem, in which case you can `export WORKING_DIR=/home/dev-scripts` and the scripts will create this directory with appropriate permissions.  In the event you create this directory manually it should be world-readable and owned by the non-root user."

We could also fix that code I linked to always chmod/chown..